var/spack/repos/builtin/packages/llvm/llvm_gcc6.patch

```diff
@@ -0,0 +1,15 @@
+--- a/projects/libcxx/include/thread
++++ b/projects/libcxx/include/thread
+@@ -435,7 +435,12 @@ sleep_for(const chrono::duration<_Rep, _Period>& __d)
+     using namespace chrono;
+     if (__d > duration<_Rep, _Period>::zero())
+     {
++#if defined(_LIBCPP_COMPILER_GCC) && (__powerpc__ || __POWERPC__) && __GNUC__ > 6
```

Comment:
The code in llvm master doesn't restrict this to gcc > 6:
```c++
#if defined(_LIBCPP_COMPILER_GCC) && (__powerpc__ || __POWERPC__)
    //  GCC's long double const folding is incomplete for IBM128 long doubles.
        _LIBCPP_CONSTEXPR duration<long double> _Max = nanoseconds::max();
#else
        _LIBCPP_CONSTEXPR duration<long double> _Max = duration<long double>(ULLONG_MAX/1000000000ULL) ;
#endif
```

Reply:
I needed that test to make it compile with gcc/6.4.0 on summit.

Reply:
On summit, I need the else to be executed otherwise it does not compile (gcc 6 and 7). So I negated the expression. Not sure why the llvm master is the way it is.